packages/experiments/src/components/Announced/Announced.base.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { BaseComponent, DelayedRender, classNamesFunction } from '../../Utilities';
+import { IProcessedStyleSet } from '../../Styling';
+import { IAnnouncedProps, IAnnouncedStyles, IAnnouncedStyleProps } from './Announced.types';
+
+const getClassNames = classNamesFunction<IAnnouncedStyleProps, IAnnouncedStyles>();
+
+export class AnnouncedBase extends BaseComponent<IAnnouncedProps, {}> {
+  public static defaultProps: Partial<IAnnouncedProps> = {
+    ariaLive: 'assertive'
+  };
+
+  private _classNames: IProcessedStyleSet<IAnnouncedStyles>;
+
+  public render(): JSX.Element {
+    const { message, styles, theme, id, ariaLive } = this.props;
+
+    this._classNames = getClassNames(styles, {
+      theme: theme!
+    });
+
+    return (
+      <div role="status" aria-live={ariaLive} id={id}>
```

Comment:
Would it be useful for developers to customize this element type perhaps with DIV as default? Not sure since the content is typically hidden, but worth consideration.